### ISSUE TITLE
Installation on Mac issues from T.Kosciolek (7th feb)

### ISSUE THREAD

**Issue by DanBuchan** [psipred/fragfold_idp#6]
### General installation issues

1. sounds good.
2. `pip install` gives me an error:
>Could not find a version that satisfies the requirement PyBrain==0.3.3 (from -r requirements.txt (line 6)) (from versions: 0.2.1, 0.3)
>No matching distribution found for PyBrain==0.3.3 (from -r requirements.txt (line 6))

    Works fine with version 0.3.

3. Shouldn't it read something along the lines of?
    Switch to your Python2 environment and install [ansible](https://github.com/ansible/ansible)

5. OK (but initially I get an error saying there is no numpy and after that pip retries and installs biopython)  

8. Is this really the only way? I pip installed ansible, so I need to find where it got installed first and then run this startup script from there. We should also change the path. Also, I'm not sure it's necessary if I do `pip install`

10. this time a got an error on unpacking BLAST+

    What actually made me realize that you assume people are running this on a Linux machine. Which is fine, but it doesn't say so anywhere. And, for example, I tried to run it on a mac, hence the error above (I guess).

### ansible `paths.yml` comments
1. why some variables have `dir` in their name and some have `path`?  

2. what are those: `hhdb` `hhdbversion` `hhpdb` `blastversion` `dynamine_api_key`
what I'm getting at, is that we should let users know that they may update some versions, i.e. `hhdb` `hhdbversion` `hhpdb`, but rather not `blastversion`.

3. Commented out section should probably read: `leave commented`, instead of `uncommented`, right?

4. `install.yml` in ansible folder also has some hard-coded (but commented out) paths. Should they be removed?

**Comment by DanBuchan** [comment]
In response to theses issues 

### General issues
2. changed pybrain to version 0.3 as suggested
3. python2 is not required to checkout ansible so only the next step requires switching to python2.
4. I didn't get that numpy error but I did discover ansible requires jinja2.  I have made the appropriate changes to the readme
5. ansible should be cloned from git not installed with pip, I have updated the readme accordingly
6. For some reason ansible does not like Mac's version of tar, I have fixed the ansible installed to handle mac or linux unpacking

### paths.yml stuff

1. because reasons. I have dropped the use of path and in fact in doing so it exposed some issues that would cause problems if people wanted to install the dependencies in different root paths. I've rewritten the paths.yml and install.yml to be more flexible (but this makes it a little more complex to edit)
2. In general users can update any of these dependencies with later versions, I have added some explanatory comments to the paths.yml
3. Correct
4. I'm leaving the commented out bits as they refer to pfclust which one day we can re-enable and also paths.yml exists so that users don't have to open install.yml so I'm leaving those commented lines

tl;dr: these things are fixed on my dev machine and will be pushed to master shortly
